backend/api.py

```diff
@@ -0,0 +1,168 @@
+from flask import Flask, request, jsonify, make_response
+from flask_sqlalchemy import SQLAlchemy
+import uuid
+from werkzeug.security import generate_password_hash, check_password_hash
+import jwt
+import datetime
+from functools import wraps
+
+app = Flask(__name__)
+
+app.config['SECRET_KEY'] = 'hierkommtnochmalwasgutesreinhaha'

```

Comment:
Weißt du ob wir hier Umgebungsvariablen nutzen können?

Dann könnten wir die in Heroku festlegen und müssen die nicht committen (besonders für Passwörter wichtig).